TombLib/TombLib/LevelData/Compilers/TombEngine/Textures.cs

```diff
@@ -89,33 +94,33 @@ private void BuildSprites()
                     var id = spriteTextureIDs[sprite.Texture.Hash];
                     if (id == -1)
                     {
-                        _progressReporter.ReportWarn("Sprite #" + sequence.Sprites.IndexOf(sprite) + 
+                        _progressReporter.ReportWarn("Sprite #" + sequence.Sprites.IndexOf(sprite) +
                             " in sequence #" + sequence.Id.TypeId + " wasn't added: size is too big or coordinates are invalid.");
                         continue;
                     }
 
-                    /*var packInfo = spriteAllocator.GetPackInfo(id);
-                 
-                    float x = packInfo.Pos.X / (float)_spritesTexturesPages[packInfo.OutputTextureID].Width;
-                    float y = packInfo.Pos.Y / (float)_spritesTexturesPages[packInfo.OutputTextureID].Height;
+                    var packInfo = spriteAllocator.GetPackInfo(id);
+
+                    int padding = 8;
```

Comment:
Why padding value is hardcoded? Can't we use padding specified in level settings?